client/python/cryoet_data_portal/src/cryoet_data_portal/_models.py

```diff
@@ -16,34 +20,34 @@
-        cell_strain_id (str): Link to more information about the cell strain
+        cell_strain_id (str): NCBI Identifier for the cell line or strain
         cell_strain_name (str): Cell line or strain for the sample.
         cell_type_id (str): Cell Ontology identifier for the cell type
         dataset_citations (str): DOIs for publications that cite the dataset. Use a comma to separate multiple DOIs.
         dataset_publications (str): DOIs for publications that describe the dataset. Use a comma to separate multiple DOIs.
-        deposition_date: Date when a dataset is initially received by the Data Portal.
+        deposition_date (date): Date when a dataset is initially received by the Data Portal.
         description (str): A short description of a CryoET dataset, similar to an abstract for a journal article or dataset.
-        funding_sources: List[FundingSource] An array relationship with FundingSource
+        funding_sources (List[DatasetFunding]): An array relationship
         grid_preparation (str): Describe Cryo-ET grid preparation.
-        https_prefix (str): The HTTPS directory path where this dataset is contained
+        https_prefix (str): The https directory path where this dataset is contained
+        id (int): An identifier for a CryoET dataset, assigned by the Data Portal. Used to identify the dataset as the directory name in data tree
         key_photo_thumbnail_url (str): URL for the thumbnail of preview image.
         key_photo_url (str): URL for the dataset preview image.
-        last_modified_date (date):Date when a released dataset is last modified.
+        last_modified_date (date): Date when a released dataset is last modified.
         organism_name (str): Name of the organism from which a biological sample used in a CryoET study is derived from, e.g. homo sapiens
         organism_taxid (str): NCBI taxonomy identifier for the organism, e.g. 9606
         other_setup (str): Describe other setup not covered by sample preparation or grid preparation that may make this dataset unique in  the same publication
         related_database_entries (str): If a CryoET dataset is also deposited into another database, enter the database identifier here (e.g. EMPIAR-11445). Use a comma to separate multiple identifiers.
-        related_database_links (str): If a CryoET dataset is also deposited into another database, e.g. EMPIAR, enter the database identifier here (e.g.https://www.ebi.ac.uk/empiar/EMPIAR-12345/).  Use a comma to separate multiple links.
+        related_database_links (str): If a CryoET dataset is also deposited into another database, e.g. EMPAIR, enter the database identifier here (e.g.https://www.ebi.ac.uk/empiar/EMPIAR-12345/).  Use a comma to separate multiple links.
         release_date (date): Date when a dataset is made available on the Data Portal.
-        runs: List[Run] An array relationship with Run
+        runs (List[Run]): An array relationship
```

Comment:
This is the description of this relationship in schema.graphql and should probably be updated there (or whatever its ultimate source is).

This is a common theme for the list and item relationships.